tools/sdk-testgen/packages/autorest.testmodeler/CHANGELOG.json

```diff
@@ -1,6 +1,18 @@
 {
```

Comment:
For my curiosity why do we have the CHANGELOG.json file as well as the MD?

Reply:
Hi @weshaggard, Both of them are generated by 'rush publish': https://rushjs.io/pages/commands/rush_publish/

I understand MD is for human reading as a summary and JSON is for detail commit history.  Both of the files are committed in the repo Azure/Autorest, so we are using the same approach here :)